SS_readcontrol_330.tpl

```diff
@@ -3111,8 +3111,12 @@
     {
       echoinput<<Comp_Err_ParmCount<<"  #_parameters are needed"<<endl;
       Comp_Err_Parm_Start=N_selparm;
-      for(f=1;f<=Comp_Err_ParmCount;f++)
-        {N_selparm++; ParCount++; ParmLabel+="ln(DM_theta)_"+NumLbl(f);}
+      for (f=1;f<=Nfleet;f++) {
+      if (DM_parmlist(f)>0) {
+       N_selparm++; ParCount++; ParmLabel+="ln(DM_theta)_Len_"+NumLbl(f)+"_P"+NumLbl(Comp_Err_L2(f));}}
```

Comment:
addition of both fleet number and parameter number in parameter label will be useful for r4ss as the status-quo system relies on reading the data file to get the assignment of parameter number to fleet (discussed in https://github.com/r4ss/r4ss/issues/287).